cypress_test/integration_tests/common/impress_helper.js

```diff
@@ -145,7 +145,6 @@ function selectTextOfShape(selectAllText = true) {
 	cy.waitUntil(function() {
 		cy.cGet('svg g .leaflet-interactive')
 			.then(function(items) {
-				expect(items).to.have.length(1);
```

Comment:
maybe we should retain this has whatever is the equivalent of "length > 0" ?

Reply:
this check has already been removed as part of https://github.com/CollaboraOnline/online/commit/b69642e9fb094b1a6a0a613554ba35d69d2746aa on master :)